crates/chia-bls/src/bls_cache.rs

```diff
@@ -64,7 +71,7 @@ impl BlsCache {
 
             // Otherwise, we need to calculate the pairing and add it to the cache.
             let mut aug_msg = pk.borrow().to_bytes().to_vec();
-            aug_msg.extend_from_slice(msg.as_ref());
```

Comment:
I would expect `extend_from_slice()` to be more efficient. Is there a good reason to make this change?